### ISSUE TITLE
Bug: Rule "Use String Templates to assemble Text" can not handle line breaks 

### ISSUE THREAD

**Issue by fmabap** [SAP/abap-cleaner#306]
Hi,

we found a bug, that the ABAP Cleaner uglifies this statement:

<img width="1244" alt="image" src="https://github.com/SAP/abap-cleaner/assets/101019006/c8702b64-6e88-4aab-8914-012c0c1c39cb">

It looks like, that it can not handle the line breaks.

We have version v1.16.1. of the ABAP Cleaner.
In the test profile is only the rule "Use String Templates to assemble Text" active with default settings:

<img width="453" alt="image" src="https://github.com/SAP/abap-cleaner/assets/101019006/97107a20-a0e1-432a-a458-57b2bbc2eb03">

We restrict the rule syntax to ABAP 7.50. But the issue occurs also with the latest ABAP release.

Here is the test code
```abap
  METHOD test.
    DATA lv_ts    TYPE char26.
    DATA lv_date  TYPE date.
    DATA lv_time  TYPE sy-uzeit.
    DATA lv_char6 TYPE char6.

    lv_char6 = '000000'.
    lv_date = sy-datum.
    lv_time = sy-uzeit.

    lv_ts = lv_date+0(4) && '-' &&
        lv_date+4(2) && '-' &&
        lv_date+6(2) && '-' &&
        lv_time+0(2) && '.' &&
        lv_time+2(2) && '.' &&
        lv_time+4(2) && '.' &&
        lv_char6(6).
  
    WRITE lv_ts.
  ENDMETHOD.
```

Can you please fix it?

Best regrads
Frank

**Comment by jmgrassau** [comment]
Hi Frank, 

thanks for reporting this bug! It should be fixed with the next release: 

![image](https://github.com/SAP/abap-cleaner/assets/79639298/6da43fd5-6388-4637-87ff-dc0d1e22189b)

Kind regards,
Jörg-Michael

**Comment by jmgrassau** [comment]
Hi Frank, 

thanks again for opening this issue – this should now be fixed with version 1.16.2, which was just released.

Kind regards,
Jörg-Michael

**Comment by fmabap** [comment]
> Hi Frank,
> 
> thanks again for opening this issue – this should now be fixed with version 1.16.2, which was just released.
> 
> Kind regards, Jörg-Michael

Hi Jörg-Michael,

thank you for the quick fix.

Kind regards
Frank
